### ISSUE TITLE
Quic connection protocol establishment error

### ISSUE THREAD

**Issue by 2ndMessiah** [SanCraftDev/cloudflared#76]
Hi Zoey&David! 

I've been using your CFd images for some time! This is rly convenient coupled with watchtower.

Appreciate it for bringing us this wonderful image and keeping it updated.

I randomly checked the container log and found some errors taking place but trivial. It looks related to the TLS encryption method when a quick connection is trying to be established. I was wondering if docker itself led to this error. The docker compose file is the one from the repo.

<img width="1215" alt="image" src="https://github.com/ZoeyVid/cloudflared/assets/22229188/40e35c8f-cbcb-4398-bbc8-165db8d6024f">

**Comment by Zoey2936** [comment]
Does it work again?

**Comment by 2ndMessiah** [comment]
> Does it work again?

Hi!! Happy to get your reply!

I searched this issue in cloudflared repo and found some results such as https://github.com/cloudflare/cloudflared/issues/1148

I think this is their bug so I closed the issue. The cloudflare devs state that they have fixed the bug in version 2024.1.2, but actually, it still exists in our image which uses the 2024.1.4 version CFD..So I'm confused.

Luckily it will automatically fall back to the http2 connection if QUIC fails, and won't actually damage the user experience AFAIK.
